pkg/cloud/services/network/subnets_test.go

```diff
@@ -224,137 +193,9 @@ func TestReconcileSubnets(t *testing.T) {
 						},
 					}),
 					gomock.Any()).Return(nil)
-
-				m.CreateTags(gomock.Eq(&ec2.CreateTagsInput{
-					Resources: aws.StringSlice([]string{"subnet-1"}),
-					Tags: []*ec2.Tag{
-						{
-							Key:   aws.String("kubernetes.io/cluster/test-cluster"),
-							Value: aws.String("shared"),
-						},
-						{
-							Key:   aws.String("kubernetes.io/role/internal-elb"),
-							Value: aws.String("1"),
-						},
-					},
-				})).
-					Return(&ec2.CreateTagsOutput{}, nil)
-
-				m.CreateTags(gomock.Eq(&ec2.CreateTagsInput{
-					Resources: aws.StringSlice([]string{"subnet-2"}),
-					Tags: []*ec2.Tag{
-						{
-							Key:   aws.String("kubernetes.io/cluster/test-cluster"),
-							Value: aws.String("shared"),
-						},
-						{
-							Key:   aws.String("kubernetes.io/role/internal-elb"),
```

Comment:
We want these tags to be tested for unmanaged subnets too after fixing the above comment.

Reply:
Done